packages/jaeger-ui/src/utils/config/get-config.tsx

```diff
@@ -26,36 +26,52 @@ let haveWarnedDeprecations = false;
  * default config from `../../constants/default-config`.
  */
 const getConfig = memoizeOne(function getConfig() {
-  const getJaegerUiConfig = window.getJaegerUiConfig;
-  if (typeof getJaegerUiConfig !== 'function') {
-    if (!haveWarnedFactoryFn) {
-      // eslint-disable-next-line no-console
-      console.warn('Embedded config not available');
-      haveWarnedFactoryFn = true;
-    }
-    return { ...defaultConfig };
-  }
-  const embedded = getJaegerUiConfig();
+  const capabilities = getCapabilities();
+  const config = {...defaultConfig, storageCapabilities: capabilities};
+
+  const embedded = getUiConfig();
   if (!embedded) {
-    return { ...defaultConfig };
+    return config;
   }
   // check for deprecated config values
   if (Array.isArray(deprecations)) {
     deprecations.forEach(deprecation => processDeprecation(embedded, deprecation, !haveWarnedDeprecations));
     haveWarnedDeprecations = true;
   }
-  const rv = { ...defaultConfig, ...embedded };
+  const rv = { ...config, ...embedded};
   // mergeFields config values should be merged instead of fully replaced
   const keys = mergeFields;
   for (let i = 0; i < keys.length; i++) {
     const key = keys[i];
-    if (typeof embedded[key] === 'object' && embedded[key] !== null) {
-      rv[key] = { ...defaultConfig[key], ...embedded[key] };
+    if (embedded && typeof embedded[key] === 'object' && embedded[key] !== null) {
+      rv[key] = { ...config[key], ...embedded[key] };
     }
   }
   return rv;
 });
 
+function getUiConfig() {
+  const getJaegerUiConfig = window.getJaegerUiConfig;
+  if (typeof getJaegerUiConfig !== 'function') {
+    if (!haveWarnedFactoryFn) {
+      // eslint-disable-next-line no-console
+      console.warn('Embedded config not available');
+      haveWarnedFactoryFn = true;
+    }
```

Comment:
```suggestion
      // eslint-disable-next-line no-console
      console.warn('Embedded config not available');
```
Can we try this? There is a unit test that checks for log-only-once and I think it will still pass because of memoization

Reply:
You are right, the memoized did the job.